start.py

```diff
@@ -287,6 +287,7 @@ def cm_server_not_dead(primary_node):
         deployment.refresh_parcel_repos()
         # Remove RC from version.
         version = args.sdc_version.rsplit('-RC')[0]
+        version = args.sdc_version.rsplit('-SNAPSHOT')[0]
```

Comment:
this i think will break for anything ending with RC .. version is set for RC then its overridden with the outcome of SNAPSHOT one. also when a version ends with other words like `latest` this will not work. maybe combine line 289 and 290 as: `'.'.join([i for i in re.split('[.-]', args.sdc_version) if i.isdigit()])`